README.md

```diff
@@ -15,8 +15,8 @@ log out and in again.
 
 ## Setup and Commands
 
-Before starting, get a copy of `Apornas ö 2020-01-11.pdf` from the Drive and save it as
-`resources/script.pdf`.
+Before starting, download all files from `Kulturkrock/Projekt/Skärmkrock/Utvecklingsdata` on
```

Comment:
My intent with this change was to upload all assets to the drive, but apparently I forgot. I've uploaded the video I used for seagull.mp4 now, do you have files for the other assets you could upload?

Reply:
No, all of mine are fakes :(